account.rs/src/wasm/src/account.rs

```diff
@@ -342,6 +345,55 @@ impl Account {
             .map_err(AccountError::AesGcmSiv)
     }
 
+    #[wasm_bindgen(js_name = decryptNaclUtxo)]
+    pub fn decrypt_nacl_utxo(
+        &self,
+        ciphertext: Vec<u8>,
+        commitment: Vec<u8>,
```

Comment:
We should add a check that commitment is of length 32 bytes and throw a meaningful error if not, since it's not plausible that a commitment hash is not 32 bytes.